### PR TITLE
Replace references to tf-cli

### PR DESCRIPTION
**Pull request by mikenomitch**

### CODE REVIEW

File: content/source/docs/enterprise/upgrade/differences.html.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ The new Terraform Enterprise does not support the `terraform push` command, whic
 
 This does not mean we're dropping support for a CLI-based workflow — we want push to be better, and the integrated `push` command was holding us back because we built it on an outdated understanding of how Terraform Enterprise should work. Users of `terraform push` have a few options:
 
-- Use [the add-on Terraform Enterprise command line tools](https://github.com/hashicorp/tfe-cli). These are designed as a one-to-one replacement for `terraform push`, with a modified syntax and some new features.
+- Use the [community supported Terraform CLI helper](https://github.com/hashicorp-community/tf-helper). This was designed as a one-to-one replacement for `terraform push`, with a modified syntax and some new features.
```
Comment (armchairlinguist) [inline content/source/docs/enterprise/upgrade/differences.html.md]: @fprimex Does this work for the new tfh? 

@mikenomitch I'm wondering if we ought to consider just removing this page, as I think it's mostly out of date now. This was useful during the time that people were transitioning to the new workflow/UI.

File: content/source/docs/enterprise/workspaces/creating.html.md [hunk 2]
```diff
@@ -39,12 +40,12 @@ VCS-backed workspaces support several optional fields, which you can reveal by c
...
 
-    -> **Note:** If you specify a working directory, TFE will still queue a plan for changes to the repository outside that working directory. This is because local modules are often outside the working directory, and changes to those modules should result in a new run. If you have a repo that manages multiple infrastructure components with different lifecycles and are experiencing too many runs, we recommend splitting the components out into independent repos. See [Repository Structure](./repo-structure.html) for more detailed explanations.
+  -> **Note:** If you specify a working directory, TFE will still queue a plan for changes to the repository outside that working directory. This is because local modules are often outside the working directory, and changes to those modules should result in a new run. If you have a repo that manages multiple infrastructure components with different lifecycles and are experiencing too many runs, we recommend splitting the components out into independent repos. See [Repository Structure](./repo-structure.html) for more detailed explanations.
```
Comment (armchairlinguist) [inline content/source/docs/enterprise/workspaces/creating.html.md]: @mikenomitch I think we'd be better to mention filtering here, at this point!

File: content/source/docs/enterprise/upgrade/differences.html.md (hunk 1 above)
Comment (fprimex) [reply content/source/docs/enterprise/upgrade/differences.html.md]: `tfh pushconfig` and `tfh pushvars` are the same as in `tfe-cli`, yep.

File: content/source/docs/enterprise/workspaces/creating.html.md (hunk 2 above)
Comment (mikenomitch) [reply content/source/docs/enterprise/workspaces/creating.html.md]: Heh, I didn't see this at all (only in the diff cause of a linter change), but I'll add something about it!

File: content/source/docs/enterprise/upgrade/differences.html.md (hunk 1 above)
Comment (mikenomitch) [reply content/source/docs/enterprise/upgrade/differences.html.md]: @armchairlinguist - yeah I wondered about removing the page too. Who would be the person/people to make that call? (hoping its you 😄 )

File: content/source/docs/enterprise/workspaces/creating.html.md (hunk 2 above)
Comment (mikenomitch) [reply content/source/docs/enterprise/workspaces/creating.html.md]: FYI, I spoke with the rest of the team and they'll handle this in a different PR/ticket.

File: content/source/docs/enterprise/upgrade/differences.html.md (hunk 1 above)
Comment (mikenomitch) [reply content/source/docs/enterprise/upgrade/differences.html.md]: FYI, I spoke with the rest of the team and they'll handle this in a different PR/ticket.

**Pull request merged**